backend/portal/api/curation/v1/curation/collections/actions.py

```diff
@@ -61,7 +60,7 @@ def post(body: dict, user: str):
     metadata = CollectionMetadata(body["name"], body["description"], body["contact_name"], body["contact_email"], links)
 
     try:
-        version = get_business_logic().create_collection(user, metadata)
+        version = get_business_logic().create_collection(user, body.get("curator_name", ""), metadata)
```

Comment:
Trying to figure out what the product requirement is. Also this code is taken from the existing code: see [here](https://github.com/chanzuckerberg/single-cell-data-portal/blob/77384dcb343d67496f5031c1556fc2b3a00b2bc8/backend/portal/api/collections_common.py#L127)

Reply:
connexion prevents curator_name from being passes in the body which is why `create_collection_common` is the way it is. Change this to:
```suggestion
        version = get_business_logic().create_collection(user, "", metadata)
```
This makes is clear that curator_name shouldn't be in the body. Otherwise we are sending mixed signals about whether curator_name should be modified by this endpoint.

Reply:
The reason we don't populate this in the curation api is because we don't have the id_token form auth0 to tell us what the user's name is. For now the frontend supplies this value when the collection is created or updated.